content/ember/v1/access-to-instances.md

```diff
@@ -48,5 +48,8 @@ App.instanceInitializer({
 
 Added in [PR #10256](https://github.com/emberjs/ember.js/pull/10256).
 
+<!-- Link isn't working? am a bit confused by repo setup?  -->
 For more information, see [the instance initializer deprecation
-guide](../instance-initializers).
+guide](/ember/instance-initializers).
```

Comment:
what's up with this link? if it's not in the `v1/` directory there is something unique with the routing? also will take a closer look